### PR TITLE
Add support for Ubuntu in ops

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: ops/inventories/production.yml [hunk 1]
```diff
@@ -9,41 +9,28 @@ all:
     app_directory: ota
     ota_repository: https://github.com/ambanum/OpenTermsArchive.git
     ota_branch: main
+    app_config: "{{ lookup('file','../config/{{config_file_name}}.json') | from_json }}"
```
Comment (martinratinaud) [inline ops/inventories/production.yml:12]: Very nice!

File: ops/roles/infra/tasks/main.yml [hunk 2]
```diff
@@ -45,17 +49,28 @@
...
   # See https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#recommended-enable-user-namespace-cloning
 - name: Enable user namespace cloning to allow running Chromium in a sandbox
   command: sysctl -w kernel.unprivileged_userns_clone=1
   when: ansible_facts['architecture'] != "aarch64"
 
 - include_tasks: mongo.yml
-  when: ansible_facts['architecture'] != "aarch64"
+  when:
+    - mongo_required
+    - ansible_facts['architecture'] != "aarch64"
```
Comment (martinratinaud) [inline ops/roles/infra/tasks/main.yml:76]: Can you please add a comment on why it should not run on aarch64.
Also, if this is a mongo version problem, can't we use mongo 4 on ubuntu?

**Review by martinratinaud — CHANGES_REQUESTED** [review]

File: ops/roles/infra/tasks/main.yml (hunk 2 above)
Comment (Ndpnt) [reply ops/roles/infra/tasks/main.yml:76]: Not really, I do not know exactly why we cannot use mongo on aarch64. By the way, it was you that added this condition, so If you remember why, please add the comment yourself :)

Comment (martinratinaud) [reply ops/roles/infra/tasks/main.yml:77]: In order to test, uncomment the lines and try to deploy on vagrant: `ansible-playbook ops/site.yml -i ops/inventories/dev.yml -vvv`

**Pull request merged**